test/dns_dnscmd/dnscmd_test.rb

```diff
@@ -153,4 +166,37 @@ def test_dnscmd_enum_zones_parses_primary_zones_only
       "domain.local",
       "TrustAnchors"], Proxy::Dns::Dnscmd::Record.new.enum_zones
   end
+
+  def test_enum_a_records
+    to_parse = '
+Returned records:
+@ 3600 A    192.168.33.33
+@ 3600 A    192.168.33.34
```

Comment:
@witlessbird - Definitely do not merge yet!  I can't reproduce this as the output format from the command!!  Really confused...